www/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
     "bluebird": "^3.4.6",
-    "gatsby": "^1.9.10",
+    "gatsby": "file:../packages/gatsby",
```

Comment:
You should use gatsby-dev-cli for copying over files during development

Reply:
yeah normally i do, but in cases where node_modules changed this ends up being easier...i wish yarn didn't save this tho.